R/calc_gsw_change.R

```diff
@@ -0,0 +1,60 @@
+#' Calculate Global Surface Water (GSW) Change
+#'
+#' The change in water occurrence intensity between the two periods is derived
+#' from homologous pairs of months (i.e. same months containing valid
+#' observations in both periods). The difference in the occurrence of surface
+#' water was calculated for each homologous pair of months. The average of all
+#' of these differences constitutes the Surface Water Occurrence change
+#' intensity. The raster files have integer cell values between [0, 200] where 0
+#' represents surface water loss and 200 represents surface water gain.
+#'
+#' The pixel values are aggregated using method provided via the
+#' \code{stats_gsw} parameter.
+#'
+#' @param x A single polygon for which to calculate the GSW statistics.
+#' @param global_surface_water_change The GSW Change data source.
+#' @param engine The preferred processing functions from either one of "zonal",
+#' "extract" or "exactextract". Default: "extract".
+#' @param stats_gsw Aggregation function with which the data are combined.
+#' Default: "mean".
+#' @return A tibble
```

Comment:
Could you please describe briefly columns names and what they mean here? (applies to all indicators)